github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -8,6 +8,7 @@ presubmits:
       testgrid-dashboards: kubevirt-presubmits
     always_run: false
     optional: true
+    cluster: prow-workloads
     skip_report: true
```

Comment:
we want skip report = true ?
optional = true is enough i think ?

Reply:
yes we want them this way, while we test them on periodic.